File: Modul - 6/Soal - 1/PRAK601-2410817220009-NABILLAPUTRINUGRAHA.c
#include <stdio.h>
int main() {
    int baris, kolom;
   
    //buat nginput baris sama kolom
    scanf("%d %d", &baris, &kolom);

    int matriks[baris][kolom];

    //buat nginput elemen matriks
    for(int i = 0; i < baris; i++) {
        for(int j = 0; j < kolom; j++) {
            scanf("%d", &matriks[i][j]);
        }
    }

    //buat nyetak matriks
    printf("\n");
    for(int i = 0; i < baris; i++) {
        for(int j = 0; j < kolom; j++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: Modul - 6/Soal - 1/PRAK601-2410817220009-NABILLAPUTRINUGRAHA.py
#fungsi buat nyetak matriks
def print_matriks(matriks):
    for baris in matriks :
        print(' '.join(map(str, baris)))
#buat nginput jumlah baris sama kolom
baris, kolom = map(int, input(" ").split())
#buat nginput elemen matriks
elemen = list(map(int, input(" ").split()))
#buat ngebuat matriks
matriks = []
for i in range(baris):
    baris = elemen[i * kolom:(i + 1)* kolom]
    matriks.append(baris)

print()
#buat nyetak baris
print_matriks(matriks)

